src/nrtk_explorer/library/dataset.py

```diff
@@ -28,8 +28,7 @@ class Dataset(TypedDict):
 
 
 def load_dataset(path: str) -> Dataset:
```

Comment:
Guess this just returns kwcoco.CocoDataset class and mypy will infure that so we can ditch the return type annotaiton and the above TypedDicts?